static/openapi/faas-spec.json

```diff
@@ -0,0 +1,670 @@
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the function versions."
+          },
+          "404": {
+            "description": "Not found."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/execute/{functionName}": {
+      "post": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Execute the latest version of a function",
+        "description": "Execute the latest version of a function for the specified function name on body.",
+        "operationId": "handle_latest_mm_execute",
+        "parameters": [
+          {
+            "name": "X-PhotonIQ-CustomerID",
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "description": "A valid JSON body is accepted",
+          "content": {
+            "application/json": {
+              "schema": {},
+              "example": {}
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Successfully executed the function."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/execute/{functionName}/{versionOrAlias}": {
+      "post": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Execute the function",
+        "description": "Execute the function for the specified function name on body.",
+        "operationId": "handle_mm_execute",
+        "parameters": [
+          {
+            "name": "X-PhotonIQ-CustomerID",
+            "in": "header",
+            "description": "The customer ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "The function name",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "versionOrAlias",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "description": "A valid JSON body is accepted",
```

Comment:
Same thing as above.  No clue what to put here.

Reply:
Here you can send any JSON your function will read

Reply:
Then you need to explain that in the API description.  I was not aware that this data was being passed to the function.